langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/googleai/PartsAndContentsMapper.java

```diff
@@ -219,9 +219,9 @@ static List<GeminiContent> fromMessageToGContent(List<ChatMessage> messages, Gem
                                 .role(GeminiRole.MODEL.toString())
```

Comment:
What about `function` role below for `TOOL_EXECUTION_RESULT` and `functionResponse`?

Please check https://ai.google.dev/gemini-api/docs/function-calling?authuser=1#expandable-7

Reply:
Unfortunately, the documentation is wrong.
I don't know why they show such a role which doesn't exist.

If ever you add a function response with that `function` role, you're going to receive an error like:
```
INVALID_ARGUMENT (code 400) Please use a valid role: user, model.
```

It's really only `user` and `model` that are allowed.

Reply:
LOL, thanks for explanation!